lists/global.csv

```diff
@@ -1234,3 +1234,11 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
 https://mitm.watch/,ANON,Anonymization and circumvention tools,2019-03-20,magma,This site detects encryption-removing MITM using logic from NDSS17 interception paper
 https://credibilitycoalition.org/,CULTR,Culture,2019-04-19,OONI,Reportedly inaccessible on WE in Egypt (but probably worth testing globally)
 https://black.riseup.net/,ANON,Anonymization and circumvention tools,2019-04-26,LEAP,
+https://gitlab.com/,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
+https://termux.org/bootstrap-x86_64.zip,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
+https://termux.net/bootstrap/bootstrap-arm.zip,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
```

Comment:
These are 5MB files which if we include as part of the global list will be fetched by every client.

Has there been an instance of this termux app being blocked in some country? Perhaps we should add these URLs to the country specific list?

Reply:
It was blocked in Iran until last month. But now on an ISP I didn't see any blockage. (Except for a lot of problems with their https communications)
It is better to test it globally since it is a world famous application. Or, if `status code: 404` is okay, I'll change it to `https://termux.net/` and `https://termux.org/` .

Reply:
How about we start off by adding it to the Iran list?

Reply:
Let's change these to the index (even if they 404) and add them inside of the Iran test list.

Reply:
OK.

Reply:
Does it makes sense to test 3 HTTPS URL of the same domain?
The 2 files of `termux.org` URLs  account for ~20M (5.8M bootstrap-arm.zip,  14M bootstrap-x86_64-v13.zip).

Reply:
@anadahz The decision changed: https://github.com/citizenlab/test-lists/pull/490